Library/.rubocop.yml

```diff
@@ -405,9 +405,16 @@ Style/NumericLiterals:
   Exclude:
     - "**/Brewfile"
 
-# OpenStruct is a nice helper.
```

Comment:
I still agree with this statement! What would you think about us talking over a refactoring some of the gnarlier uses here e.g. `cli/args` before we make a decision here?

Reply:
That or the `Tab` class which also inherits from `OpenStruct`.

Reply:
Yes, good call @apainintheneck.